src/components/pt-element/PtElementInfo.test.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import PtElementInfo from "./PtElementInfo";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+const IRON_ELEMENT = {
```

Comment:
Can we just import one of the `json` files instead? Would make keeping it in sync a bit easier.

Reply:
Sure!